include/uv_mbed/um_http.h

```diff
@@ -73,6 +74,23 @@ typedef enum http_request_state {
     completed
 } http_request_state;
 
+/**
+ * @brief HTTP responce object passed into #um_http_resp_cb.
+ */
+typedef struct um_http_resp_s {
+    um_http_req_t *req;
+
+    char http_version[4];
+    int code;
+    char *status;
+
+    int nh;
+    um_http_hdr *headers;
```

Comment:
`um_header_list` was fine for the response header (easier to deal with in Swift than the array I expect). Can deal with it as an array, but linked list slightly preferable